Formula/hblock.rb

```diff
@@ -0,0 +1,17 @@
+class Hblock < Formula
+  desc "Adblocker that creates a hosts file from multiple sources"
+  homepage "https://github.com/hectorm/hblock"
```

Comment:
maybe `https://hblock.molinero.dev/`?
```suggestion
  homepage "https://hblock.molinero.dev/"
```

Reply:
Sure, that website is also fine.